pytab/core.py

```diff
@@ -361,6 +362,13 @@ def cli(
     click.echo("| Obtaining System Information...", nl=False)
     system_info = hwi.get_system_info()
     click.echo(" success!")
+    click.echo("| Detected System Config:")
+    click.echo(yaml.dump(system_info, default_flow_style=False))
+    sysinfo_input: str = click.prompt("Is this correct? [Y/n]", type=str, default='y', show_default=False, show_choices=False)
+    if sysinfo_input is not None and sysinfo_input.lower() == 'n':
```

Comment:
Instead of checking wether it is NOT NOT FILLED, you might instead just check If its filled. 
I would suggest `if sysinfo_input and [...]`